src/models/constructor-items/sub-blocks.ts

```diff
@@ -57,6 +58,17 @@ export interface IconWrapperProps {
     icon?: PositionedIcon;
 }
 
+export enum ImageCardMargins {
+    None = 'none',
```

Comment:
do we really need the `none` parameter? We can just have a default state (now is `none`) and change it somehow when there is a demand for `s` or `m`